src/version.c

```diff
@@ -4723,11 +4723,21 @@ highest_patch(void)
     int
 has_patch(int n)
 {
-    int		i;
+    int		h, m, l;
 
-    for (i = 0; included_patches[i] != 0; ++i)
-	if (included_patches[i] == n)
+    // Perform a binary search.
+    l = 0;
+    h = (int)(sizeof(included_patches) / sizeof(included_patches[0])) - 1;
+    while (l < h)
+    {
+	m = (l + h) / 2;
+	if (included_patches[m] == n)
 	    return TRUE;
+	if (included_patches[m] < n)
+	    h = m;
+	else
+	    l = m + 1;
+    }
```

Comment:
It would be better to use the standard C function `bsearch()` to perform a binary search.

Reply:
Why would it be better to use bsearch()?  It seems invoking the compare function adds overhead.